modules/nw-route-specific-annotations.adoc

```diff
@@ -70,6 +70,34 @@ This value is applicable to re-encrypt and edge routes only. For more informatio
 Environment variables cannot be edited.
 ====
 
+.Router timeout variables
+
+`TimeUnits` are represented by a number followed by the unit: `us` *(microseconds), `ms` (milliseconds, default), `s` (seconds), `m` (minutes), `h` *(hours), `d` (days).
+
+The regular expression is: [1-9][0-9]*(`us`\|`ms`\|`s`\|`m`\|`h`\|`d`)
+[cols="3*"]
+|===
+| `ROUTER_BACKEND_CHECK_INTERVAL` | `5000ms` | Length of time between subsequent liveness checks on backends.
+| `ROUTER_CLIENT_FIN_TIMEOUT` | `1s` | Controls the TCP FIN timeout period for the client connecting to the route. If the FIN sent to close the connection does not answer within the given time, HAProxy closes the connection. This is harmless if set to a low value and uses fewer resources on the router.
+| `ROUTER_DEFAULT_CLIENT_TIMEOUT` | `30s` | Length of time that a client has to acknowledge or send data.
+| `ROUTER_DEFAULT_CONNECT_TIMEOUT` | `5s` | The maximum connection time.
+| `ROUTER_DEFAULT_SERVER_FIN_TIMEOUT` | `1s` | Controls the TCP FIN timeout from the router to the pod backing the route.
+| `ROUTER_DEFAULT_SERVER_TIMEOUT` | `30s` | Length of time that a server has to acknowledge or send data.
+| `ROUTER_DEFAULT_TUNNEL_TIMEOUT` | `1h` | Length of time for TCP or WebSocket connections to remain open. If you have websockets or TCP connections and any time HAProxy reloads, the old HAProxy processes will stay for that period.
+| `ROUTER_SLOWLORIS_HTTP_KEEPALIVE` | `300s` | Set the maximum time to wait for a new HTTP request to appear. If this is set too low, it can cause problems with browsers and applications not expecting a small `keepalive` value. See note below for more information.
+| `ROUTER_SLOWLORIS_TIMEOUT` | `10s` | Length of time the transmission of an HTTP request can take.
+| `RELOAD_INTERVAL` | `5s` | Allows the minimum frequency for the router to reload and accept new changes.
+| `ROUTER_METRICS_HAPROXY_TIMEOUT` | `5s` | Timeout for the gathering of HAProxy metrics.
+
+|===
+
+[NOTE]
```

Comment:
Is this note only really applicable to the `ROUTER_SLOWLORIS_HTTP_KEEPALIVE` row? (I see you have "See note below..."

If so, you could put this note directly in the cell for the description of that field. You'd just need to add an "a" to the column to tell it to use AsciiDoc formatting in that third column, like:

`[cols="2,1,2a", options="header"]`

Reply:
Addressed.
Thank you @bergerhoffer for all the suggestions!